src/hotspot/share/cds/cdsConfig.hpp

```diff
@@ -207,9 +207,8 @@ class CDSConfig : public AllStatic {
   static void stop_dumping_full_module_graph(const char* reason = nullptr) NOT_CDS_JAVA_HEAP_RETURN;
   static void stop_using_full_module_graph(const char* reason = nullptr) NOT_CDS_JAVA_HEAP_RETURN;
 
-  static bool is_valhalla_preview() {
-    return Arguments::enable_preview() && EnableValhalla;
-  }
+  // EnablePreview legacy
+  static bool is_valhalla_preview() { return Arguments::enable_preview(); }
```

Comment:
You might want to change all is_valhalla_preview() in CDS to just Arguments::enable_preview().  Or not.  Not completely necessary I guess.